README.md

```diff
@@ -9,3 +9,11 @@
 
 > Animals are put into an environment during a certain time(day/night) and the time will either strengthen or weaken certain traits of animals.
 
+**How it will work**
+1. Randomly select Environment(Forest, Desert)
+2. Randomly select a situation
+3. If situation = a and environment = b, then compare the selected Animals’ traits x, y, z.
+4. If Animal(1or2)’s trait x> Animal(1or2)’s trait x, give Animal(1or2) 1 point
```

Comment:
Lanuage matters, be explicit and clear... so clear that even your parents would understand :).
```suggestion
4. Let the two player's animals be represented by A and B (either order).
5. For each trait x:
5.1 If A’s trait x > B's trait x then give A 1 point added
5.2 otherwise if B's traiit x is > A's trait x then give B 1 point added
5.3 otherwise neither A nor B gets any points added
6. The winner is the animal with more points.
```